great_expectations/agent/actions/data_assistants/utils.py

```diff
@@ -47,7 +47,11 @@ def build_action_result(
     data_assistant_result: DataAssistantResult,
     id: str,
 ) -> ActionResult:
-    expectation_suite_name = f"{event.type} {event.data_asset_name} assistant suite"
+    if event.expectation_suite_name is not None:
+        expectation_suite_name = event.expectation_suite_name
+    else:
+        expectation_suite_name = f"{event.type} {event.data_asset_name} assistant suite"
```

Comment:
do we want the event type to be in this name? That seems like a implementation detail that doesn't help the user much - i'm imagining this creating a suite named something like `missingness_data_assistant_request.received orders assistant suite`.

Reply:
Hmmm good point I'll update